src/components/People/People.tsx

```diff
@@ -0,0 +1,146 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { NavLink, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+const getPersonClass = (person: Person) =>
+  classNames('', { 'has-text-danger': person.sex === 'f' });
+
+const getPersonLink = (personName: string, personBorn: number) =>
+  `${personName.toLowerCase().split(' ').join('-')}-${personBorn}`;
+
+export const People = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [peopleWithParents, setPeopleWithParents] = useState<Person[]>([]);
+  const { personName } = useParams();
+
+  const normalizePersonName = personName || '';
+
+  useEffect(() => {
+    setLoading(true);
+
+    const fetchPoeple = async () => {
```

Comment:
There is a typo in the function name `fetchPoeple`. It should be corrected to `fetchPeople`.